examples/java/src/main/java/org/apache/beam/examples/complete/TopWikipediaSessions.java

```diff
@@ -136,6 +141,20 @@ public void processElement(ProcessContext c, BoundedWindow window) {
     }
   }
 
+  static class ParseTableRowJson implements SerializableFunction<String, TableRow> {
+    private final ObjectMapper mapper =
+        new ObjectMapper().disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
```

Comment:
Yes, basically I mean use the JsonFactory (which is an indirect wrapper around ObjectMapper with standard options set) instead of the coder.

```java
import com.google.api.client.json.JsonFactory;
import com.google.api.client.json.jackson2.JacksonFactory;
...
  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
...
  TableRow row = JSON_FACTORY.fromString(string, TableRow.class)
  String string = JSON_FACTORY.toString(row);
```

But honestly, writing it out it's not much simpler (juts ensures "standard" Json configuration). Feel free to leave as-is.